api/HTMLIFrameElement.json

```diff
@@ -1079,6 +1079,8 @@
       },
       "srcdoc": {
         "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLIFrameElement/srcdoc",
+          "spec_url": "https://html.spec.whatwg.org/multipage/iframe-embed-object.html#attr-iframe-srcdoc",
```

Comment:
This is the content attribute, not the IDL attribute.

Reply:
Thanks. Fixed in next commit.